modules/internal/String.chpl

```diff
@@ -578,7 +741,15 @@ module String {
       return ret;
     }
 
-    proc strip(chars: string, leading=true, trailing=true) : string {
+    /*
+      :arg chars: A string containing each character to remove. Defaults to `" \t\r\n"`.
+      :arg leading: Indicates if leading occurrences should be removed. Defaults to `true`.
+      :arg trailing: Indicates if trailing occurrences should be removed. Defaults to `true`.
+
+      :returns: A new string with all occurrences of characters in `chars`
+                removed as indicated by `leading` and `trailing`.
```

Comment:
This sentence is a little confusing.  I think what you meant was "A new string with all occurrences of characters in `chars` removed, including `leading` and `trailing` occurrences as appropriate" or something along those lines.  Without the other parts of this documentation, one would be tempted to think the sentence you originally wrote meant that only leading and trailing occurrences were removed.